pkg/operator/staticpod/controllers.go

```diff
@@ -47,13 +50,23 @@ func NewControllers(targetNamespaceName, staticPodName string, command, revision
 		staticPodName,
 		revisionConfigMaps,
 		revisionSecrets,
-		command,
+		installerCommand,
 		kubeInformersNamespaceScoped,
 		staticPodOperatorClient,
 		kubeClient,
 		eventRecorder,
 	)
 
+	controller.pruneController = prune.NewPruneController(
+		targetNamespaceName,
+		staticPodName,
+		prunerCommand,
+		kubeClient,
+		failedRevisionLimit,
```

Comment:
just hard code this in the operator for now.  Later you can pull it from the API